src/main/kotlin/no/nav/familie/ba/sak/behandling/BehandlingService.kt

```diff
@@ -97,6 +98,28 @@ class BehandlingService(private val behandlingRepository: BehandlingRepository,
                         behandlingId)
             }
 
+    fun hentDataForKonsistensavstemming(): List<PerioderForBehandling> {
+        val relevanteBehandlinger = behandlingRepository.finnSisteIverksatteBehandlingFraLøpendeFagsaker()
+        return relevanteBehandlinger
+                .map { behandlingId ->
+                    val andelerIRelevantBehandling = beregningService.hentAndelerTilkjentYtelseForBehandling(behandlingId)
+                    andelerIRelevantBehandling
+                            .groupBy { it.kildeBehandlingId }
+                            .map { (kildeBehandlingId, andeler) ->
+                                PerioderForBehandling(behandlingId = kildeBehandlingId.toString(),
+                                                      perioder = andeler
+                                                              .map {
+                                                                  it.periodeOffset
+                                                                  ?: error("Andel ${it.id} på iverksatt behandling på løpende fagsak mangler periodeOffset")
+                                                              }
+                                                              .toSet())
+                            }
+                }
+                .reduce(::samleFagsaker)
+    }
+
+    private fun samleFagsaker(akkumulert: List<PerioderForBehandling>,
+                              nesteFagsak: List<PerioderForBehandling>) = listOf(akkumulert, nesteFagsak).flatten()
```

Comment:
Her går det vel å bruke flatten direkte? 
```suggestion
                .flatten()
    }
```

Reply:
ah, ja det kan man jo! :)